src/formio/components/story-util.js

```diff
@@ -0,0 +1,16 @@
+import { Form } from '@formio/react';
+
+export const FormioForm = ({ form }) => (<Form form={form} />);
+
+export const FormioComponent = ({ component }) => (
+  <Form form={{
+    type: 'form',
+    components: [component]
+  }}/>);
+
+export const FormDecorator = (Story) => (
+  <div
+    className="utrecht-theme utrecht-document"
```

Comment:
For the future - what would be _really_ cool is a storybook addon that lets you select the theme in a dropdown so we can toggle between them right in the canvas/docs.

Reply:
Awesome, I was hoping we could to do that too. Would you be okay with having `@utrecht/design-tokens` and `@gemeente-denhaag/design-tokens-components` in `devDependencies`?

Reply:
go ahead!